enclave/core/sgx/report.c

```
@@ -210,6 +210,9 @@ oe_result_t oe_get_remote_report(
 
     sgx_quote = (sgx_quote_t*)report_buffer;
 
+    if (sgx_quote == NULL)
+        OE_RAISE(OE_UNEXPECTED);
```

We should raise `OE_INVALID_PARAMETER` instead of unexpected